config/checkstyle/checkstyle.xml

```diff
@@ -43,6 +43,10 @@
     
     <property name="severity" value="warning"/>
 
+    <module name="SuppressionFilter">
+        <property name="file" value="${config_loc}/suppressions.xml"/>
```

Comment:
Is this necessary? If they're just warnings, I wouldn't bother with all this. I don't think we're going to be able to really go all out with CheckStyle in the foreseeable future.

Reply:
I think it is useless to check style for auto generated code. These (a lot) noise make useful warning unseen. so I'd like to exclude it.

Reply:
OK, I'd like to start another PR/thread for discussion about this.